src/test/java/org/terasology/genome/GeneIndexGenomeMapTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.genome;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+
+import com.google.common.base.Function;
+import org.terasology.genome.breed.BreedingAlgorithm;
+import org.terasology.genome.breed.MonoploidBreedingAlgorithm;
+import org.terasology.genome.breed.mutator.GeneMutator;
+import org.terasology.genome.breed.mutator.VocabularyGeneMutator;
+import org.terasology.genome.genomeMap.GeneIndexGenomeMap;
+import org.terasology.registry.In;
+import org.terasology.world.WorldProvider;
+
+import javax.annotation.Nullable;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+/**
+ * Initial test setup based on working of Genomes before trait specific breeding algorithms were introduced
+ * this test is now obsolete/serves no purpose
+ */
+class GeneIndexGenomeMapTest {
+    @In
+    WorldProvider worldProvider;
+
+    @In
+    GenomeRegistry genomeRegistry;
```

Comment:
I'm surprised that this works (using `@In` annotations for tests o.O Did you actually run this, or are the fields used yet?

Reply:
I tried using it initially and it did not work. Took out the usage but forgot to remove the declarations, fixed :)